v4/source/users.yaml

```diff
@@ -105,7 +105,16 @@
         - name: sort
           in: query
           description: |
-            Can be "", "last_activity_at" or "create_at". When left blank sorting is done by username. Must be used in conjunction with `in_team` and no other query parameters (exlcuding the paging parameters). **Minimum server version: 4.0**
+            Sort is only available in conjunction with certain options, exlcuding the paging parameters
```

Comment:
I'm not sure I understand what this means. Can it or can't it be used with paging?

Reply:
plus the spelling (which was inherited). I will attempt to reword to be clearer